templates/SearchForm.php

```diff
@@ -18,13 +18,17 @@
 		</select>
 	</label>
 	<label>Keywords
-		<input type="search" name="query" value="<?= Formatter::EscapeHtml($query ?? '') ?>"/>
+		<input type="search" name="query" placeholder="Quotes allowed, e.g., &quot;war and peace&quot;" value="<?= Formatter::EscapeHtml($query ?? '') ?>"/>
```

Comment:
Let's not include this placeholder, since that's standard search syntax and we're improving search anyway.

Reply:
Done.